Makefile

```diff
@@ -428,3 +428,6 @@ operator-sdk: $(OPERATOR_SDK)  ## Download operator-sdk locally if necessary
 $(OPERATOR_SDK):
 	curl --silent --show-error --location --fail "https://github.com/operator-framework/operator-sdk/releases/download/v1.10.1/operator-sdk_linux_amd64" --output $(OPERATOR_SDK)
 	chmod +x $(OPERATOR_SDK)
+
+run-scorecard-tests: bundle ## Run the scorecard tests
+	$(OPERATOR_SDK) scorecard bundle
```

Comment:
There is a --wait-time duration parameter.  It currently defaults to 30s.  Can we change it to 2m?  I'm concerned about slow environments.